.golangci.yml

```diff
@@ -0,0 +1,19 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  gocyclo:
+    min-complexity: 10
+  maligned:
+    suggest-new: true
+  dupl:
+    threshold: 100
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+
+linters:
+  enable-all: true
+  disable:
+    - maligned
```

Comment:
Curious, if `maligned` is disabled, why is it configured above?

Reply:
ah, I commented below, but this is actually what upstream golangCI runs on their code, so this is actually just the reference configuration. This certainly could be pruned, but I figured I'd start with upstream reference and go from there